superset/views/core.py

```diff
@@ -3001,7 +3003,7 @@ def sqllab(self):
         return self.render_template(
             "superset/basic.html",
             entry="sqllab",
-            bootstrap_data=json.dumps(d, default=utils.json_iso_dttm_ser),
+            bootstrap_data=json.dumps(d, default=utils.pessimistic_json_iso_dttm_ser),
```

Comment:
Feels weird to be passing variant/loosely-typed data to this but I guess Superset does that a lot so I won't complain much other than to point it out. :-)